lib/metadata_json_deps/runner.rb

```diff
@@ -20,12 +20,21 @@ def run
 
       @updated_module = @updated_module.sub('-', '/')
       message = "Comparing modules against *#{@updated_module}* version *#{@updated_module_version}*\n\n"
+      if check_deprecated(@updated_module_version)
```

Comment:
This allows you to override the deprecation notice when you use `@updated_module_version`. For example if I pass in `elasticsearch/elasticsearch` (a deprecated module) and `10.0.0`, then the deprecation notice won't be triggered. Replacing the argument here with a call to the Forge for the real module version would probably work here.

Reply:
Here is checked the updated module version - the typed one in command line. 
The checking for the real version of modules from the list is performed on line 36 ( message += "The compared module #{module_name.upcase} is DEPRECATED.\n" if check_deprecated(@forge.get_current_version(module_name)))